testcontainers/src/core/image.rs

```diff
@@ -249,6 +249,13 @@ impl<I: Image> RunnableImage<I> {
 }
 
 impl<I: Image> RunnableImage<I> {
+    pub fn with_args(self, args: I::Args) -> Self {
```

Comment:
Let's add a documentation?

Reply:
Sure! Gonna commit asap😁

Reply:
done, what do you think about the examples? @DDtKey

Reply:
Looks good to me, however I think the more common use-case would be a builder-like usage, i.e:

```rs
let args = vec!["arg1".to_string(), "arg2".to_string()];
let runnable_image = RunnableImage::from(image.clone())
   .with_args(args.clone());
```

This is generally a more concise syntax

Reply:
updated with that syntax! I like it more as you proposed, sorry for the delay😁

Reply:
Fixed dprint check issues in the CI, passing all checks now :smiley: